### PR TITLE
Repositioned RangeControl tooltip and adjusted image zoom control dropdown height

### LINKED ISSUE #27261: RangeControl's tooltip overlaps label
**Describe the bug**
When `RangeControl` has a visible label, the tooltip from the slider overlaps the label.

**To reproduce**
1. Add an image to an image block
2. Select the crop option
3. Select the zoom option
4. Observe the tooltip overlaps the label.

**Expected behavior**
The tooltip is below the slider

**Screenshots**
Screenshot 2020-11-25 at 4 52 01 pm

(The screenshot also shows the separate issue with the zero width slider https://github.com/WordPress/gutenberg/issues/27126)

### PR DESCRIPTION
**Pull request by Tonnie-Exelero**
## Description
- Repositioned RangeControl tooltip to the bottom so that it doesn't obscure labels.
- Adjusted Image block zoom control dropdown height to accommodate RangeControl tooltip's new position.

## How has this been tested?
- I've tested the changes on Chrome and simulated different devices and it works as intended.
- I've run unit tests.

## Screenshots 
Screenshot 2020-11-30 at 03 18 14

## Types of changes
This is a Bug Fix that introduces changes that affect the positioning of the RangeControl's tooltip to prevent it from obscuring labels. 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR. 

Fixes [#27261](https://github.com/WordPress/gutenberg/issues/27261)

### CODE REVIEW

File: packages/block-library/src/image/editor.scss [hunk 1]
```diff
@@ -107,6 +107,7 @@ figure.wp-block-image:not(.wp-block) {
 	.components-popover__content {
 		overflow: visible;
 		min-width: 260px;
+		min-height: 110px;
```
Comment (talldan) [inline packages/block-library/src/image/editor.scss:110]: The increased size doesn't look quite right when the input is focused and the tooltip isn't visible:
Screenshot 2020-12-04 at 5 12 38 pm

My feeling is that increasing the size of the popover isn't needed, it looks ok to me when the tooltip is overlapping the edge of the popover:
Screenshot 2020-12-04 at 5 09 54 pm

File: packages/components/src/range-control/styles/range-control-styles.js [hunk 2]
```diff
@@ -232,16 +232,16 @@ const tooltipShow = ( { show } ) => {
...
+	if ( isBottom ) {
 		return css`
-			top: -80%;
+			bottom: -80%;
 		`;
 	}
 
 	return css`
-		bottom: -80%;
+		top: -80%;
```
Comment (talldan) [inline packages/components/src/range-control/styles/range-control-styles.js:244]: I'm not sure all of the changes here are needed. It looks to me like it should be possible to add a `position="bottom"` prop to `SimpleTooltip` here:
https://github.com/WordPress/gutenberg/blob/149502cc880594ba3608538c57145ac586d24bf3/packages/components/src/range-control/index.js#L258-L265

That gets passed through to `Tooltip` and then to the `useTooltipPosition` hook, and finally on to `tooltipPosition`.

File: packages/components/src/range-control/tooltip.js [hunk 3]
```diff
@@ -49,17 +49,17 @@ export default function SimpleTooltip( {
...
 			let nextPosition = positionProp;
 
 			if ( positionProp === 'auto' ) {
-				const { top } = inputRef.current.getBoundingClientRect();
-				const isOffscreenTop = top - TOOLTIP_OFFSET_HEIGHT < 0;
+				const { bottom } = inputRef.current.getBoundingClientRect();
+				const isOffscreenBottom = bottom - TOOLTIP_OFFSET_HEIGHT < 0;
 
-				nextPosition = isOffscreenTop ? 'bottom' : 'top';
+				nextPosition = isOffscreenBottom ? 'top' : 'bottom';
```
Comment (talldan) [inline packages/components/src/range-control/tooltip.js:62]: Similar to the previous comment, I'm not sure this is needed if `position` is added as a prop to `SimpleTooltip`. It looks like this hook bypases all the logic in the inner `if` statement if `positionProp` has a value other than `auto`.

**Review by talldan — COMMENTED** [review]
Thanks for your work on this @Tonnie-Exelero.

I think this change should be possible with fewer lines of code, but I wasn't all that familiar with the code before this review, so I'm happy to be corrected on that 😃.

I'll add a couple of other reviewers who've modified the code recently.

File: packages/block-library/src/image/editor.scss (hunk 1 above)
Comment (Tonnie-Exelero) [reply packages/block-library/src/image/editor.scss:110]: Great, I concur @talldan. I'm removing the height adjustment in a moment.

File: packages/components/src/range-control/styles/range-control-styles.js (hunk 2 above)
Comment (Tonnie-Exelero) [reply packages/components/src/range-control/styles/range-control-styles.js:244]: Right, this makes sense.

File: packages/components/src/range-control/tooltip.js (hunk 3 above)
Comment (Tonnie-Exelero) [reply packages/components/src/range-control/tooltip.js]: Do we really need an `auto` position? I don't see the need there. The only positions making sense to me are `bottom` and `top`. I stand corrected though.

**Review by mkaz — APPROVED** [review]
Tested this using Storybook and tests well the tooltip position works as described and looks better than a label.  The previous feedback was also addressed.

👍 Thanks for the contribution!

**Comment by jeffpaul** [comment]
@Tonnie-Exelero pinging you again to see if you can help confirm your WordPress.org username so you can be properly credited in the WordPress 5.9 release? Thanks!

**Pull request merged**